### ISSUE TITLE
数据同步失败

### ISSUE THREAD

**Issue by quelang9** [Nightonke/CoCoin#13]
错误信息：
错误码   code ： 101
错误信息  ：object not found for UploadInfo.

这个数据同步功能，先查询 UploadInfo 这个表，但是bmob后台除了自带的user表，没有其他的表。那么这么查询会总是失败吧。
是不是需要配置什么？
求解答，谢谢！

**Comment by Nightonke** [comment]
Yep, you need to create every table in Bmob corresponding all the class that extends BmobObject in models. And then, create every columns as the variables.

**Comment by quelang9** [comment]
Thanks
